Application aware intentions scenario (#251)
hashicorp/katakoda: consul-application-aware-intentions/finish.md

## Review

> In this hands-on lab, you operated a Consul datacenter, deployed using Docker, to configure application aware intentions.

Reviewer: ```suggestion
In this hands-on lab, you created a Consul datacenter, deployed using Docker, and configured application aware intentions.
```
